frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.mdx

```diff
@@ -0,0 +1,104 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import { Heading, Paragraph } from '@digdir/design-system-react';
+import * as StudioTableLocalPaginationStories from './StudioTableLocalPagination.stories';
+
+<Meta of={StudioTableLocalPaginationStories} />
+
+<Heading level={1} size='small'>
+  StudioTableLocalPagination
+</Heading>
+<Paragraph>StudioTableLocalPagination is a wrapper for StudioTableRemotePagination.</Paragraph>
+<Paragraph>
+  <ul>
+    <li>
+      Use this component when you are fetching a complete set of data, and want the component to
+      handle pagination and sorting internally. Otherwise, use `StudioTableRemotePagination`.
+    </li>
+    <li>
+      Pagination and sorting can be removed by not passing props to `pagination` and setting
+      `isSortable` to `false`.
+    </li>
+  </ul>
```

Comment:
What do you think about refactoring the content to use the StudioTable component? Then we could have one column for the prop and one with the description. 

| Prop        | Description                                                                                           |
| -----------| ----------------------------------------------------------------------------------------------------- |
| isSortable | Indicates whether the component is sortable. Defaults to true.                                        |
| pagination | Defines pagination settings for the table. If the object is not provided, pagination is hidden.       |

Reply:
Very good idea, but it looked a little messy (I think the two tables are competing for attention here):
![image](https://github.com/Altinn/altinn-studio/assets/148075168/026e573b-6124-47f5-bfee-5d675a0e96ba)

Maybe it's better to list the props in the Storybook UI, like this?
![image](https://github.com/Altinn/altinn-studio/assets/148075168/8cc1dcf9-25d8-4741-97d4-6e68f1dfd8e2)